modules/geo-layers/bundle/index.js

```diff
@@ -0,0 +1,12 @@
+const deckGLLayers = require('../src');
```

Comment:
Nit: I guess these could just be `src/bundle.js`, save us a directory for just one file?

Reply:
The `core` module bundle contains multiple files.
Files under `src/` are transpiled and distributed, which is a little weird.